src/transaction/atomic_replication_helper.cpp

```diff
@@ -891,6 +917,21 @@ namespace cublog
     buf_len -= written;
   }
 
+  void
+  atomic_replication_helper::atomic_log_sequence::atomic_log_entry::dump_to_stream (
+	  std::stringstream &dump_stream) const
+  {
+    constexpr int BUF_LEN_MAX = UCHAR_MAX;
+    char buf[BUF_LEN_MAX];
+    char *buf_ptr = buf;
+    int buf_len = BUF_LEN_MAX;
+
+    // maybe faster to first dump to stack buffer
+    dump_to_buffer (buf_ptr, buf_len);
+
+    dump_stream << (char *)buf; // dump to buffer already ends with newline
```

Comment:
```suggestion
    dump_stream << (char *) buf; // dump to buffer already ends with newline
```

Reply:
Should be done by the indent tool. Probably not done because it's `xpp' file.